corporate/views/portico.py

```diff
@@ -143,9 +145,16 @@ def communities_view(request: HttpRequest) -> HttpResponse:
             )
             unique_org_type_ids.add(realm.org_type)
 
+    # Custom list of org types to show.
+    ORG_TYPES_TO_SHOW = [
```

Comment:
Let's call this `CATEGORIES_TO_OFFER` -- we're showing all organizations regardless of whether their type is one of these.

Reply:
Done.